Allow ocsigenserver.2.5 to build against newer Lwt.

mfp opened the pull request:
Trivial 1-line changes to build against newer Lwt where `Lwt_util` and `Lwt_event` have been removed.

dsheets commented:
On OCaml 4.01.0:

```
# ocamlfind: Package `tyxml.parser' not found
# ocamlfind: Package `tyxml.parser' not found
# ocamlfind: Package `tyxml.parser' not found
# ocamlfind: Package `tyxml.parser' not found
# ocamlfind: Package `tyxml.parser' not found
# ocamlfind: Package `lwt.syntax' not found
```

mfp commented:
On Sun, Apr 26, 2015 at 06:51:56AM -0700, David Sheets wrote:
> On OCaml 4.01.0:
> 
> ```
> # ocamlfind: Package `tyxml.parser' not found
> # ocamlfind: Package `tyxml.parser' not found
> # ocamlfind: Package `tyxml.parser' not found
> # ocamlfind: Package `tyxml.parser' not found
> # ocamlfind: Package `tyxml.parser' not found
> # ocamlfind: Package `lwt.syntax' not found

Reproduced and fixed in pristine 4.01.0 switch, waiting for Travis to confirm.
I guess the CI wasn't working at the time this was packaged because there's no
way it could have passed in 4.01.0 (missing camlp4 hard dependency).

Also fixed other issue with newer Lwt when sqlite3 is installed and
ocsigenserver tries to build the corresponding ocsipersist back-end.

-- 
Mauricio Fernández
